spec/components/access_panels/at_the_library_component_spec.rb

```diff
@@ -116,6 +116,10 @@
       render_inline(described_class.new(document:))
     end
 
+    it 'has solr document location_display of nil' do
+      expect(document.location_display).to be_nil
```

Comment:
The function is used in the component and that data used there is another good place to make sure everything is working correctly. The solr document has a physical location so that is why I put it in there.